src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -570,7 +577,7 @@ class module_elf : public module_impl
           auto symbol_type = static_cast<patcher::symbol_type>(rela->r_addend);
           std::vector<uint64_t> offsets;
           offsets.push_back(offset);
-          arg2patchers.emplace(std::move(argnm), patcher{ symbol_type, std::move(offsets) });
+          arg2patchers.emplace(std::move(argnm), patcher{ symbol_type, std::move(offsets), buf_type });
```

Comment:
Maybe this will work without the local offsets variable and push back above?
```suggestion
          arg2patchers.emplace(std::move(argnm), patcher{ symbol_type, {offset}, buf_type });
```

Reply:
Maybe this will work. But it has been this way for a while, And I do not see significant improvement using the suggested change either.